### ISSUE TITLE
[Nit] Consider un-rotating figures in the text

### ISSUE THREAD

**Issue by mciura** [jemmybutton/byrne-euclid#15]
IMHO, all non-trivial figures in the text can be shown in the same orientation as in the accompanying picture and startGlobalRotation can be used only for trivial figures: points, lines, arcs.  See the triangle in Proposition I.II, the angles in Proposition I.VIII, the rectangle in Proposition II.V, etc.  The only counterargument I can think of is consistency with Byrne's book.

**Comment by jemmybutton** [comment]
I agree, sometimes it's absolutely unnecessary (angles in prop:I.VIII look especially strange), but sometimes it makes sense, like [here](https://archive.org/stream/firstsixbooksofe00eucl#page/258/mode/2up), for example: rotated rectangles are much more compact and look quite nice (to me) while still recognizable. And there are some cases in between (the triangle in prop:I.II seem to look better when standing on its base, but why not rotate the whole picture to make the triangle be still on its base while being rotated the same way as it is on the main picture). I do most of the experimentation in the Russian version and, as it deviates more from the original, I start with removing unnecessary complications there ([this time](https://github.com/jemmybutton/byrne-euclid/commit/e211ea02ad618f9149ffb43027d43ee5ead43fff), that stupid thing with angles and the triangle in prop:I.II). The rectangle and that gnomon-like thing in prop:II.V are [unrotated](https://github.com/jemmybutton/byrne-euclid/commit/eedbe73162648e68d04187b08b465a1db6409b7f) now in both versions (the main picture is rotated differently that in the original anyway).
In general, I want to keep the English version closer to the original when it doesn't make much harm, but in the Russian version I try to get rid of most obvious excesses (for example, two angle styles in the Byrne's book are used only once [one here](https://archive.org/stream/firstsixbooksofe00eucl#page/24/mode/2up) and [and one here](https://archive.org/stream/firstsixbooksofe00eucl#page/126/mode/2up), they are currently present in the English version, but they are replaced with more generic ones in the Russian translation).
To sum up: thank you for this suggestion, it makes sense, I'll start with cleaning the Russian version in that respect and then I'll either make some simplified version of the English one, or will just clean the main one (to remove stuff is much easier than to add it back in this case).
